supportedBackends/google/batch/src/test/scala/cromwell/backend/google/batch/api/BatchRequestExecutorSpec.scala

```diff
@@ -238,13 +251,44 @@ class BatchRequestExecutorSpec
     result.status match {
       case RunStatus.Failed(_, events, _) =>
         val eventNames = events.map(_.name)
-        val eventTimes = events.map(_.offsetDateTime.toString)
-        println(eventNames)
```

Comment:
I'm guessing this is a debug line that should be removed?

Reply:
Yes. Maybe this is a leftover from previous PR. The test do succeed without the println.

Reply:
Oh, yes, right, I got confused and though it was being added here 🤦‍♀️